### DISCUSSION TITLE
Prevent json frontmatter from being processed

### DISCUSSION THREAD

**Discussion by kaosine** [prettier/prettier#13534] [Q&A] [answered]
We have this implemented with our vuepress site, but every time it runs it turns this front-matter to a single line. Such as:

```json
---
{ "title": "Page title here", "lang": "en-us" }
---
```
There has to be something we can implement, so even on a short one it doesn't do that. That's just awful the way it looks and makes it a pain when we need to add in the future...

**Comment by fisker** [comment, marked as answer]
You can disable it by `embeddedLanguageFormatting` https://prettier.io/docs/en/options.html#embedded-language-formatting
